src/api/api.py

```diff
@@ -0,0 +1,34 @@
+from .pros import pros
+from flask import Flask, jsonify, abort
+from werkzeug.exceptions import HTTPException
+from flask_cors import CORS
+
+cors = CORS()
+
+api = Flask(__name__)
+
+cors.init_app(api)
```

Comment:
Move initialization of `cors` right about `cors.init_app()`. That way, there's less chance of those getting separated in the future if more code gets added after _api_ initialization.